### ISSUE TITLE
Issue Displaying Spell Components and Passive Skills not showing when enabled.

### ISSUE THREAD

**Issue by AkimboGogurts** [ElfFriend-DnD/foundryvtt-compactBeyond5eSheet#69]
Hello! I recently installed this module and am having a couple problems:

I'm Running:

Foundry VTT Version: 10 Build: 291
DnD5e : Version 2.0.3

When viewing the spell list of my players i see this:

![image](https://user-images.githubusercontent.com/30539535/206060911-c440a779-2fc9-4dca-b6c4-c33dd4b87643.png)

I've tried a few things to narrow this down. 

- Resynchronized PC sheet from D&D Beyond - no change
- Draw in a spell from the built in D&D Spell compendium - no change
- Draw in a spell from the D&D Beyond Importer Compendium - no change
- Remove Spell Components text from spell - no change
- Revert to Default character sheet (with action list still active) - shows list without [object, Object]
- Started a new World and fresh PC sheet with only this module and the action list enabled, same issue. 

My second issue is that Passive Scores don't appear when I check them in the Module options:

![image](https://user-images.githubusercontent.com/30539535/206061417-8f30c298-3f83-455a-a6e6-25768f98296d.png)

Thank you for any help!

**Comment by AkimboGogurts** [comment]
Apologies I should clarify the Other,other thing is appearing in the spell list of the sheet, not the action list.

**Comment by AkimboGogurts** [comment]
Hello! A little extra information: 

I tried reverting my version to the oldest version of Foundry 10 I could grab (10.282), and the spellbook list continues to show the [Object,Object] entries. 

I did a little digging in the element inspector and here is what it looks like:

![image](https://user-images.githubusercontent.com/30539535/207199203-a1829433-8e51-4f28-831b-819aef517fdc.png)

Editing the spell and unchecking the spell component boxes (verbal, somatic) makes the letters go away along with a single instance of the [Object, Object] Entry.
